src/Microsoft.AspNet.Mvc.Core/MvcBuilderExtensions.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNet.Builder
     /// <summary>
     /// Extension methods for <see cref="IApplicationBuilder"/> to add MVC to the request execution pipeline.
     /// </summary>
-    public static class BuilderExtensions
+    public static class MvcBuilderExtensions
```

Comment:
We should call this `MvcCoreApplicationBuilderExtensions` or something even longer and more horrid. We're about to add an `IMvcBuilder` so this name is a little ambiguous. Also, there's 'MVC' extension methods for `IApplicationBuilder` coming from a few packages (hence the 'Core').